src/detail-page/DetailPage.tsx

```diff
@@ -20,27 +19,48 @@ export const TabContainer = styled('div')`
   }
 `;
 
+const getData = (professionId: ProfessionKey) => {
```

Comment:
Could we add return type for this function?

Reply:
Done